pkgs/os-specific/linux/trezor-udev-rules/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "trezor-udev-rules";
+  version = "unstable-2019-07-17";
+
+  udevRules = fetchurl {
+    # let's pin the latest commit in the repo which touched the udev rules file
+    url = "https://raw.githubusercontent.com/trezor/trezor-firmware/54d348228f1187f356f3ae7f1fcc618243108121/common/udev/51-trezor.rules";
```

Comment:
Just a minor comment - there is a newer version (whitespace fix) of this file https://raw.githubusercontent.com/trezor/trezor-firmware/0ee0ade761211047ed30e0dc851e60a8a323c876/common/udev/51-trezor.rules

Reply:
Addressed in d124858efebc9b027f271d75cc32523dca84485a